### ISSUE TITLE
color picker not losing focus on mouse leaving editor window

### ISSUE THREAD

**Issue by awalone995** [godotengine/godot#45138]
**Godot version:**
3.2.4beta6 mono

**OS/device including version:**
win7

**Issue description:**
so you see the picker is always almost at the top right there

![tttqB8WkOd](https://user-images.githubusercontent.com/26692947/104362827-55689400-5525-11eb-98f2-46cb1de188f2.gif)

the focus keeps even if i release the mouse after exiting the window so you have to aim for that gap to set smh like 255-128-128

tldr :: it's impossible to quickly set a top color without ruining the selection **sEVERAL TIMES** 
it's such a minor thing that probably requires changes in the editor core but jesus christ i cant stand it any more

**Steps to reproduce:**
am not sure if it's os specific

**Comment by Calinou** [comment]
Duplicatge of https://github.com/godotengine/godot/issues/35185.
